odoo/addons/base/models/ir_actions.py

```diff
@@ -82,14 +82,15 @@ def _get_eval_context(self, action=None):
         }
 
     @api.model
-    @tools.ormcache('frozenset(self.env.user.groups_id.ids)', 'model_name')
-    def get_bindings(self, model_name):
+    @tools.ormcache('frozenset(self.env.user.groups_id.ids)', 'model_name', 'debug')
+    def get_bindings(self, model_name, debug=False):
```

Comment:
Why do we need to add this debug attribute?

Reply:
`base.group_no_one` is special group which activated on debug mode and mostly we manage it manually like this: 
https://github.com/odoo/odoo/blob/master/odoo/models.py#L1150

get_bindings is works well with all groups except `base.group_no_one` and it is not calculated again due to ormcache, so we need to pass/process it manually.
Exactly like menus: https://github.com/odoo/odoo/blob/master/odoo/addons/base/models/ir_ui_menu.py#L78